quarkus/config-api/src/main/java/org/keycloak/config/LoggingOptions.java

```diff
@@ -11,16 +13,27 @@ public class LoggingOptions {
     public static final Output DEFAULT_CONSOLE_OUTPUT = Output.DEFAULT;
     public static final String DEFAULT_LOG_FILENAME = "keycloak.log";
     public static final String DEFAULT_LOG_PATH = "data" + File.separator + "log" + File.separator + DEFAULT_LOG_FILENAME;
+    public static final Boolean GELF_ENABLED = isGelfEnabled();
```

Comment:
Isn't better to run this code only during build time? Not sure how much it will save (Quarkus CL is indexed) but one less call at runtime.

If this code is not running during static init at runtime, please ignore it.

Reply:
@pedroigor Thanks for the review. 

I was thinking about it, as I was also curious about the execution time of it :) You're right that it's executed during static init at runtime, but the execution time of the check itself when GELF is present is ~300μs, and when it is not, it's ~200μs. It's really low, and also the static init evaluation it's negligible in this case. Thus, I decided to use this approach as I didn't want to create another logic around the build process.

@pedroigor But if you think it's a blocker for this PR, we can figure it out.

Reply:
Changes like that are tricky because they work both ways with little impact at runtime, as you pointed out. The tricky part comes when we are adding small changes like that and their sum ends up introducing a considerable time penalty.

I would say we should not neglect start-up time/memory footprint (which is already a bit higher, I think) and try to remove/move operations that can be avoided at runtime.

Reply:
@pedroigor Ok, fair enough.

Reply:
@pedroigor Is this a blocker from your PoV? Would it work for you to address this as a follow-up?

Reply:
I'd also like to address that in a follow-up task because it might be a little bit out of the scope of this PR due to the absence of support for obtaining persisted property in `config-api` module. I don't see any big deal to providing the property such as is ATM. 

@pedroigor Can we do it later?